Core/Base/PluginManager.php

```diff
@@ -305,6 +305,7 @@ private function getPluginInfo($pluginName, $iniContent)
 
             if ($info['min_version'] >= 2018 && $info['min_version'] <= self::MIN_VERSION) {
                 $info['compatible'] = true;
+                $info['description'] = self::$i18n->trans('compatible');
```

Comment:
Small rule of thumb, always that you add a new translation key, it should be added at least in spanish and english locale files, it will make more ease to keep the l18n files updated

Reply:
I know that, I found a lot of missing translation in past that I add after browsing from new feature added. This time is my fault ;)